src/components/Facility/AssetCreate.tsx

```diff
@@ -473,10 +449,10 @@ const AssetCreate = (props: AssetProps) => {
         className="grow-0 pl-6"
         crumbsReplacements={{
           [facilityId]: {
-            name: locationsQuery.data?.results[0].facility?.name,
+            name,
           },
           assets: { style: "text-secondary-200 pointer-events-none" },
-          [assetId || "????"]: { name },
+          [assetId || "????"]: { name: name || undefined },
```

Comment:
We don't want to display undefined in case name is not available; replace it with "Asset"